examples/practice-typescript/src/components/Button/index.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../styles/bases/variable.scss';
+
+.btn {
+  letter-spacing: 0;
+  color: $color-text-secondary;
+  border: 0.067rem;
+  border-radius: 1.15rem;
+
+  &__add {
```

Comment:
This is scss syntax. As discuss we use css only for this practice. So, you could use CSS nested https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting

```css
.btn {
  letter-spacing: 0;
 /**/

  .btn__add {}
```